theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/Marker.tsx

```diff
@@ -0,0 +1,83 @@
+import type {Pointer} from '@theatre/dataverse'
+import {useVal} from '@theatre/react'
+import {pointerEventsAutoInNormalMode} from '@theatre/studio/css'
+import getStudio from '@theatre/studio/getStudio'
+import {lockedCursorCssPropName} from '@theatre/studio/uiComponents/PointerEventsHandler'
+import useContextMenu from '@theatre/studio/uiComponents/simpleContextMenu/useContextMenu'
+import useRefAndState from '@theatre/studio/utils/useRefAndState'
+import React from 'react'
+import styled from 'styled-components'
+import {attributeNameThatLocksFramestamp} from '@theatre/studio/panels/SequenceEditorPanel/FrameStampPositionProvider'
+import type {SequenceEditorPanelLayout} from '@theatre/studio/panels/SequenceEditorPanel/layout/layout'
+
+const Mark = styled.div<{left: number}>`
```

Comment:
Suggestion: Try out a `IMarkThemeValues` type to see what it looks like, maybe? I don't feel strongly, but as you add colors it could make more sense.

Reply:
`left` was refactored out